### PR TITLE
Make `Latest blocks` and `Adjustments` widget title clickable

### PR DESCRIPTION
**Pull request by nymkappa**
Second attempt 👍🏻 

Fixes https://github.com/mempool/mempool/issues/1862 @Xekyo

#### Main dashboard

Screen Shot 2022-06-14 at 10 36 21 AM

#### Mining dashboard

Screen Shot 2022-06-14 at 10 36 52 AM

### CODE REVIEW

**Review by wiz — APPROVED** [review]
Tested ACK @ v2.4.1-dev [[0409c9a9](https://github.com/mempool/mempool/commit/0409c9a9)]

Will wait for @softsimon to review before merging

**Review by softsimon — APPROVED** [review]
utACK

**Pull request merged**
